src/action/common/create_group.rs

```diff
@@ -141,13 +153,16 @@ impl Action for CreateGroup {
                 patch: _,
             }
             | OperatingSystem::Darwin => {
-                execute_command(Command::new("/usr/bin/dscl").args([
-                    ".",
-                    "-delete",
-                    &format!("/Groups/{name}"),
-                ]))
-                .await
-                .map_err(|e| CreateGroupError::Command(e).boxed())?;
+                // TODO(@hoverbear): Make this actually work...
+                // Right now, our test machines do not have a secure token and cannot delete users.
```

Comment:
Maybe have some kind of output for a user running this that this operation is not currently implemented?

Reply:
Great idea!